src/query/react/buildHooks.ts

```diff
@@ -112,6 +124,19 @@ interface UseQuerySubscriptionOptions extends SubscriptionOptions {
   refetchOnMountOrArgChange?: boolean | number
 }
 
+/**
+ * A React hook that automatically triggers fetches of data from an endpoint, and 'subscribes' the component to the cached data.
```

Comment:
```suggestion
 * A React hook that automatically triggers a request for a given endpoint, and 'subscribes' the component to the cached data.
```

Reply:
I think this copy would be slightly more readable and matches more closely to the `UseMutation` language below.